lib/mosaic-utils/src/main/java/org/eclipse/mosaic/lib/util/VehicleUtils.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2024 Fraunhofer FOKUS and others. All rights reserved.
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contact: mosaic@fokus.fraunhofer.de
+ */
+
+package org.eclipse.mosaic.lib.util;
+
+import org.eclipse.mosaic.lib.geo.CartesianPoint;
+import org.eclipse.mosaic.lib.math.Vector3d;
+import org.eclipse.mosaic.lib.math.VectorUtils;
+
+public class VehicleUtils {
```

Comment:
Two remarks:

1. Are we satisfied with the naming and package here? All calculations are somewhat related to the geomath module. The class-name is really broad for what it's doing atm. Maybe something along the lines of `VehicleReferenceUtils`
2. If we provide wrapper for `Vector3d` and `CartesianPoint` should we not also just add `GeoPoint`?

Reply:
1. Good point, naming it to `VehicleReferenceUtils` is a good idea. I would like to leave it in mosaic-utils though, as it is already quite specific within the MOSAIC context, whereas the classes in mosaic-geomath are targeting more general math calculations.
2. Will do